src/Raven.Server/Web/Authentication/AdminCertificatesHandler.cs

```diff
@@ -525,6 +527,8 @@ public Task GetCertificates()
                         writer.WriteComma();
                         writer.WritePropertyName("LoadedServerCert");
                         writer.WriteString(Server.Certificate.Certificate?.Thumbprint);
+                        writer.WriteComma();
+                        writer.WriteArray("WellKnownCerts", wellKnown);
```

Comment:
I would change that to `WellKnownAdminCerts`

Reply:
done